www/pages/docs/layouts.md

```diff
@@ -83,13 +83,13 @@ const store = createStore(
 
 // Next we need to import a list of files that will be generated when Greenwood is run
 import '../index/index.js';
-import './list';
 
 // Finally we can connect to our store and define our component.
 connectRouter(store);
 
-// Add a \`MYROUTES\` predefined hook. This is where all your routes will be loaded.
+// Add the MY-ROUTES(without dash) predefined hook. This is where all your routes will be loaded.
```

Comment:
Why not just keep without the dash in the comment too?  Also, maybe ton confuse hook with our plugins, let's call it a constant?

so all together would be:
> `// Add the `MYROUTES` predefined constant. This is where all your routes will be loaded.`

Reply:
I had to add dash intentionally otherwise it will be caught by our find and replace within scaffold lifecycle which breaks compilation.